tests/integration/ha_tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+import pytest as pytest
+from pytest_operator.plugin import OpsTest
+
+from tests.integration.ha_tests.helpers import (
+    app_name,
+    change_master_start_timeout,
+    get_master_start_timeout,
+)
+
+APPLICATION_NAME = "application"
+
+
+@pytest.fixture()
+async def continuous_writes(ops_test: OpsTest) -> None:
+    """Deploy the charm that makes continuous writes to PostgreSQL."""
+    # Deploy the continuous writes application charm if it wasn't already deployed.
+    async with ops_test.fast_forward():
+        if await app_name(ops_test, APPLICATION_NAME) is None:
+            charm = await ops_test.build_charm("tests/integration/ha_tests/application-charm")
```

Comment:
Perhaps these test `application-charm`s could be renamed to be more descriptive? not essential for this PR though.

Reply:
Godo suggestion! I created a ticket for that on Jira.